browser/js/signup/signup.html

```diff
@@ -12,19 +12,37 @@
     </div>
     <div class="form-group">
         <label for="email">Email address</label>
-        <input type="text" name="email" ng-model="signup.email" class="form-control" id="email" placeholder="Enter email">
+        <input type="text" name="email" ng-model="signup.user.email" class="form-control" id="email" placeholder="Enter email">
     </div>
     <div class="form-group">
         <label for="signup-password">Password</label>
-        <input type="password" name="password" ng-model="signup.password" class="form-control" id="signup-password" placeholder="Password">
+        <input type="password" name="password" ng-model="signup.user.password" class="form-control" id="signup-password" placeholder="Password">
     </div>
     <div class="form-group">
         <label for="signup-firstName">First Name</label>
-        <input type="firstName" name="firstName" ng-model="signup.firstName" class="form-control" id="signup-firstName" placeholder="First Name">
+        <input type="firstName" name="firstName" ng-model="signup.user.firstName" class="form-control" id="signup-firstName" placeholder="First Name">
     </div>
     <div class="form-group">
         <label for="signup-lastName">Last Name</label>
-        <input type="lastName" name="lastName" ng-model="signup.lastName" class="form-control" id="signup-lastName" placeholder="Last Name">
+        <input type="text" name="lastName" ng-model="signup.user.lastName" class="form-control" id="signup-lastName" placeholder="Last Name">
+    </div>
+    <div class="form-group">
+        <label for="signup-streetAddress">Street Address</label>
+        <input type="text" name="streetAddress" ng-model="signup.address.address" class="form-control" id="signup-streetAddress" placeholder="Street Address">
+    </div>
+    <div class="form-group">
+        <label for="signup-city">City</label>
+        <input type="text" name="city" ng-model="signup.address.city" class="form-control" id="signup-city" placeholder="City">
+    </div>
+    <div class="form-group">
+        <label for="signup-zipCode">State</label>
+        <select ng-model="signup.address.state">
+            <option ng-repeat="state in states">{{state}}</option>
+        </select>
+    </div>
+    <div class="form-group">
+        <label for="signup-zipCode">Zip Code</label>
+        <input type="text" name="zipCode" ng-model="signup.address.zipCode" class="form-control" id="signup-zipCode" placeholder="Zip Code">
     </div>
```

Comment:
I think you already mentioned it, but we should add that all fields are required.  Check out the address-form directive - you can just add an ng-disabled on the submit button and make every input required.